test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -9,14 +9,10 @@ Describe "Select-String" -Tags "CI" {
 	    { $testinputone | Select-String -Pattern "hello" } | Should Not Throw
 	}
 
-	it "Should be called without error using the sls alias" {
-	    { $testinputone | sls -Pattern "hello" } | Should Not Throw
-	}
-
-    it "Should return an array data type when multiple matches are found" {
-        $result = $testinputtwo | Select-String -Pattern "hello"
-        ,$result | Should BeOfType "System.Array"
-    }
```

Comment:
The first - Previous test do that so the test is alias related only.
The second is not removed - it is below - diff trick 😕